src/System.Net.Requests/tests/System.Net.Requests.Tests.csproj

```diff
@@ -47,7 +47,6 @@
     <Compile Include="HttpWebRequestHeaderTest.cs" />
     <Compile Include="HttpWebRequestTest.netstandard.cs" />
     <Compile Include="HttpWebResponseHeaderTest.cs" />
-    <Compile Include="HttpRequestResponseSerializationTest.cs" />
     <Compile Include="$(CommonTestPath)\System\Runtime\Serialization\Formatters\BinaryFormatterHelpers.cs">
```

Comment:
We should remove the BinaryFormatterHelpers.cs from csproj as well, suspect it's used elsewhere other than the removed file. Don't require it in compilation.